tests/licensedcode/data/false_positive/false-positive-gpl3.txt

```diff
@@ -0,0 +1 @@
+S5PC100_GPL3(0)
```

Comment:
Can you check out the text in https://github.com/aboutcode-org/scancode-toolkit/issues/3932#issue-2550246988, I believe we wanted the whole highlighted text there as test